src/triage/cli.py

```diff
@@ -436,5 +437,32 @@ def history(self, args):
         db_history(dburl=self.root.db_url)
 
 
+@Triage.register
+class SavePredictions(Command):
```

Comment:
Might suggest renaming this to avoid confusion since there's a separate CLI argument `save-predictions` that can be used when running the experiment. Perhaps something like `AddPredictions` or something similar?